### PR TITLE
Uvpx

### PR DESCRIPTION
**Pull request by KotlinIsland**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -16,6 +16,22 @@ requires = [
 ]
 build-backend = "setuptools.build_meta"
 
+
+[tool.pyprojectx]
+main = ["uv"]
+
+[tool.pyprojectx.aliases]
+install = "uv pip install test-requirements.txt"
```
Comment (DetachHead) [inline pyproject.toml:24]: -e .

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,58 +1,41 @@
...
+clone the project using your IDE
 
-#### (3) Create then activate a virtual environment
-
-```bash
-python3 -m venv venv
-source venv/bin/activate
+if you don't use an IDE, you can also use the terminal (not recommended, the terminal is better left in the 80's):
+```shell
+git clone git@github.com:<your_username>/basedmypy.git
```
Comment (DetachHead) [inline CONTRIBUTING.md:22]: is this using the ssh method (never works) instead of the https method?

**Review by DetachHead — APPROVED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (KotlinIsland) [reply CONTRIBUTING.md:22]: trueeee

**Comment by KotlinIsland** [comment]
wheres the .vscode folder? fail

**Pull request merged**